### ISSUE TITLE
Suggestion - custom furniture presets

### ISSUE THREAD

**Issue by scufflord** [InZidiuZ/op-framework-issue-tracker#3101]
### Assurance

I have read the `README.md`

### Summary

Hey I'm Scufflord with Scufflord's Shitty Middle of the Night Idea Emporium and I'm here to tell you about my new idea: furniture presets. Do you have a Michael's mansion decorated the exact way you want, but your realtor told you they have to drop the block? Worried about all that furniture that you spent hours playing Sims 4: GTA edition customizing just going away forever? Fear not, this github idea will solve those issues.

Basically, allow people to save 3-5 "presets" of furniture configurations with some kind of command. Then, when you purchase a new house or if an interior decorator wants to just decorate with a known good design for someone, they can just move the furniture in with one press of the button.

### Reason

Make Furniture Script Great Again. I think it would really help out a lot with interior decorating businesses etc.

**Comment by coalaura** [comment]
I'm not that big of a fan of this idea. I feel like it would make it boring and remove the uniqueness of each house if you just copy paste the same design over and over again
